html/user/server_status.php

```diff
@@ -183,6 +185,11 @@ function show_status_html($x) {
         ";
     }
     end_table();
+
+    if ( isset($server_version) && isset($git_commit) ) {
+        echo "Server version: $server_version ($git_commit)\n";
```

Comment:
Maybe have this a hyperlink to `https://github.com/BOINC/boinc/commit/$git_commit` ?